### ISSUE TITLE
Cannot see the Getting Started video in Full Screen

### ISSUE THREAD

**Issue by vkanne-msft** [openT2T/opent2t.github.io#7]
On both Chrome and Edge i cannot get the control for Full Screen on the getting started video.

**Comment by tjaffri** [comment]
This is really annoying... sorry. The issue is with the player itself on channel9, not the embed code on this site.

For example, click these two links:
1. https://channel9.msdn.com/Blogs/Seth-Juarez/IoT/player (will show full screen button)
2. https://channel9.msdn.com/Blogs/Open-Translators-to-Things/Open-Translators-to-Things-Introduction-for-Users/player (will not show full screen button)

I thought it was because I published these videos as a series, rather than a blog but I've updated them to be in a blog (to match the first link) but they still don't work.

I think the videos actually need to be re-uploaded, and perhaps there are some settings as part of the upload process I need to fix. Will try this when I get a chance... or will talk to channel9.

Thanks for reporting the issue!
